ApollosProject/apollos-templates
# chore: Stop running migrations automatically, run migrations by hand.

Steps to test
1. Make sure you are pointed at your local database schema (not production) 
2. From the API folder, run `yarn migrator down` 
3. Boot the API, look for a special message telling you that you have migrations to run
4. Run `yarn migrator up`. 
5 Confirm that migrations were ran. 

Also note commands like `yarn migrator --help`
